webcompat/form.py

```diff
@@ -34,11 +34,15 @@
 
 problem_choices = [
     (u'detection_bug', u'Desktop site instead of mobile site'),
-    (u'mobile_site_bug', u'Mobile site is not usable'),
-    (u'video_bug', u'Video doesn\'t play'),
-    (u'layout_bug', u'Layout is messed up'),
-    (u'text_bug', u'Text is not visible'),
-    (u'unknown_bug', u'Something else - I\'ll add details below')
+    (u'mobile_site_bug', u'Site is not usable'),
+    (u'layout_bug', u'Design is broken'),
+    (u'video_bug', u'Video or audio doesn\'t play'),
```

Comment:
@miketaylr if I change that input value to 'media_bug', then one of the current functional tests breaks

https://github.com/webcompat/webcompat.com/blob/master/tests/functional/reporting-non-auth.js#L258

is this going to cause some domino effect where the media type reporting tool will also need to be changed? ex this is the resulting url when I use the media type reporting tool for new issue:

https://webcompat.com/issues/new?url=http%3A%2F%2Fsafeschool.mon.bg%2FVideos%2F201705210928251574.mp4%3FAspxAutoDetectCookieSupport%3D1&label=type-media&problem_type=video_bug&src=media-decode-error&details=Technical+Information%3A%0AError+Code%3A+NS_ERROR_DOM_MEDIA_METADATA_ERR+%280x806e0006%29%0AResource%3A+http%3A%2F%2Fsafeschool.mon.bg%2FVideos%2F201705210928251574.mp4